cns/restserver/restserver.go

```diff
@@ -8,6 +8,7 @@ import (
 	"fmt"
```

Comment:
should we just introduce an api to create a network?

Reply:
I don't think a new api is needed here. This will be only be used by orchestrators on windows only so commandline flag should be sufficient in my opinion. Avoiding introducing new api.